.neutrinorc.js

```diff
@@ -52,20 +142,19 @@ module.exports = {
           },
         },
       },
-      eslint: {
-        rules: {
-          // This has been set in the latest Airbnb preset, but has not been
-          // released yet.
-          'react/no-did-mount-set-state': 'off',
-        }
-      }
+      env: [
+        'APPLICATION_NAME',
+        'LOGIN_STRATEGIES',
+        'PORT',
+        'TASKCLUSTER_ROOT_URL',
+      ],
+      babel: {
+        plugins: [
+          [require.resolve('@babel/plugin-proposal-decorators'), { legacy: true }],
+          require.resolve('@babel/plugin-proposal-class-properties'),
+        ],
+      },
     }],
-    ['@neutrinojs/env', [
-      'NODE_ENV',
-      'APPLICATION_NAME',
-      'LOGIN_STRATEGIES',
-      'TASKCLUSTER_ROOT_URL',
-    ]],
     (neutrino) => {
       if (process.env.NODE_ENV === 'development') {
         neutrino.config.devtool('cheap-module-source-map');
```

Comment:
I'd move this to the `devtool` option within `@neutrinojs/react`, since it ensures terser-webpack-plugin will be configured correctly (switching to terser is currently unreleased; see https://github.com/neutrinojs/neutrino/pull/1158).

Also, the default is `cheap-module-eval-source-map`, which is likely faster for incremental builds (given eval), so unless there is some issue being encountered, might be best left at the default. See:
https://webpack.js.org/configuration/devtool/#devtool